index.html

```diff
@@ -2866,9 +2866,17 @@ <h3>Profiles</h3>
       These actions can be done by anyone without specific training.
       </p>
       <p>
-        In the core profile  we define a mapping to
-        HTTP(S). It is envisioned, that future versions of this document
-        contain additional mapping to Web Sockets and other protocols.
+        In the HTTP baseline profile we define a mapping to
```

Comment:
not sure if the use of `we` is used elsewhere in the document. I would prefer the passive structure